test/transforms/test_replicate.py

```diff
@@ -1,14 +1,14 @@
+import os
 from itertools import count
 from pprint import pprint
-from unittest.mock import Mock
 
 import pytest
-from requests import HTTPError
+from taskcluster.exceptions import TaskclusterRestFailure
 from taskgraph.util.templates import merge
 
 from mozilla_taskgraph.transforms.replicate import transforms as replicate_transforms
 
-TC_ROOT_URL = "https://tc-tests.example.com"
+TC_ROOT_URL = os.environ.get("TASKCLUSTER_PROXY_URL", "https://tc-tests.example.com")
```

Comment:
Is there a reason we need to support overriding this in the environment? These get mocked out anyways, so there should never be a reason for them to be a real proxy URL?

(If this were non-test code this would probably be needed...but in test code I can't see why we'd want it.)

Reply:
Ah; in that case I think it would be preferable to forcibly override this environment variable, to ensure that we're always using the value specified here. (With the current code, I would expect this to be the _real_ tc proxy when running in tc. This probably wouldn't cause issues most of the time, but eventually I imagine it will cause some confusion.)